sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -235,18 +236,18 @@ object ScalaNativePluginInternal {
             logger.running(compilec)
             val result = Process(compilec, cwd) ! logger
             if (result != 0) {
-              println("Failed to compile native library runtime code.")
+              sys.error("Failed to compile native library runtime code.")
```

Comment:
Note that `sys.error` will be deprecated in 2.13: https://github.com/scala/scala/pull/5677. It's better to use `throw new RuntimeException` instead.